components/axonaut/axonaut.app.mjs

```diff
@@ -0,0 +1,97 @@
+import { axios } from "@pipedream/platform";
+
+export default {
+  type: "app",
+  app: "axonaut",
+  propDefinitions: {
+    companyId: {
+      label: "Company ID",
+      description: "The company ID",
+      type: "string",
+      async options() {
+        const companies = await this.getCompanies();
```

Comment:
I am wondering if the API provides pagination for get companies. Looks like we are fetching everything or just the first page.

Reply:
I didn't find pagination in the docs.